fetch-utils.js

```diff
@@ -36,7 +36,30 @@ export async function logout() {
 
     return (window.location.href = '../');
 }
+//--------------------------------------------------------------------------------------------
+export async function completeItem(id) {
+    const response = await client.from('groceries').update({ complete: true }).match({ user_id: client.auth.user().id, id: id });
+    return checkError(response);
+}
+
+export async function getItems() {
+    const response = await client.from('groceries').select().order('complete').match({ user_id: client.auth.user().id });
+    console.log(response);
+    return checkError(response);
+}
+
+export async function createItem(item) {
+    const response = await client.from('groceries').insert({ description: item, complete: false, user_id: client.auth.user().id });
+    console.log(response, 'testing');
+    return checkError(response);
```

Comment:
In the future, we can invoke our checkAuth function where createItem() is called to match the user_id. Definitely not a requirement, nice work!